src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -168,11 +173,16 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
     [],
   )
 
+  const hasEnoughStakingBalance = useMemo(
+    () => bnOrZero(cryptoAmount).lte(amountAvailableCryptoPrecision),
+    [amountAvailableCryptoPrecision, cryptoAmount],
+  )
```

Comment:
Exposed at component-level too so we can disable the query in case of no-ops.

This was the explanation for "Amount too low" when inputting *more* than your staked balance previously. The ratio we were doing to get the bps was resulting in > 10_000 bps, which makes sense since input/staked amount including fees is a positive ratio.